src/languages/ko-KR.js

```diff
@@ -254,7 +254,7 @@ export default {
     octopushTypePremium: "프리미엄 (빠름) - 알림 기능에 적합해요)",
     octopushTypeLowCost: "저렴한 요금 (느림) - 가끔 차단될 수 있어요)",
     "Check octopush prices": "{0}에서 Octopush 가격을 확인할 수 있어요.",
-    octopushPhoneNumber: "휴대전화 번호 (intl format, eg : +33612345678) ",
+    octopushPhoneNumber: "휴대전화 번호 (intl format, 예시: +82 1023456789) ",
```

Comment:
```suggestion
    octopushPhoneNumber: "휴대전화 번호 (intl format, 예시: +821023456789) ",
```

Doesn't library require non-space input?

Reply:
I tried to make it look better by using spacing.

Reply:
It could lead a user to enter input as wrong format. I have checked the other languages and they choose non-space.